src/main/java/duke/Event.java

```diff
@@ -0,0 +1,18 @@
+package duke;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends duke.Task {
+
+    private String date;
+    private DateTimeFormatter test;
+
+    public Event(String description, String date) {
+        super(description);
+        this.date = date;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at:" + date + ")";
```

Comment:
Would it be better to change the `[E]` string to a named constant? I noticed this for the other classes that extend from Task as well.